client/my-sites/site-settings/style.scss

```diff
@@ -339,6 +339,12 @@
 			vertical-align: bottom;
 			color: $alert-green;
 		}
+
+		.gridicons-notice-outline {
```

Comment:
Since these styles are the same for `.gridicons-checkmark`, can we combine them and just keep the `color` separate?